### PR TITLE
fix: when dbclick Preferences did not prevent

### PR DESCRIPTION
**Pull request by GideonSenku**

### CODE REVIEW

File: src/entrypoints/ui/utils/figma.ts [hunk 1]
```diff
@@ -7,5 +7,8 @@ export function getCanvas() {
 
 export function getObjectsPanel() {
   // Similar to `getCanvas()`.
-  return document.querySelector('[data-testid="objects-panel"]') as HTMLElement
+  const node =
+    document.querySelector('[data-testid="objects-panel"]') ||
+    document.querySelector('[data-testid="devHandoffNodesList"]')
```
Comment (Justineo) [inline src/entrypoints/ui/utils/figma.ts:12]: Is this from Dev Mode? Are we expecting using the extension along with Dev Mode itself? I think most features are already included in Dev Mode.

Comment (GideonSenku) [reply src/entrypoints/ui/utils/figma.ts:12]: sorry, this is for development, because Figma remove the `window.figma`, if u mind, i can remove it

Comment (Justineo) [reply src/entrypoints/ui/utils/figma.ts:12]: OK let's remove it for now. You can now run the extension on a editable file in design mode to test.

**Comment by Justineo** [comment]
Thanks!

**Pull request merged**